src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import cn from 'classnames';
+import { Link } from 'react-router-dom';
+import { Tab } from '../../types/Tab';
+
+interface Props {
+  tabs: Tab[];
+  tabId: string;
+}
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  tabId,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === tabId);
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              className={cn({ 'is-active': selectedTab?.id === tab.id })}
```

Comment:
Maybe it would be better to create a variable here

Reply:
Thank you so much!)